### PR TITLE
updated parallelism changes

### PR DESCRIPTION
**Pull request by karthikaselvaraj-okta**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,7 +5,13 @@ aliases:
   - &build_steps
     - checkout
     - run: java -version
-    - run: ./mvnw clean install -Pci
+    - run: mvn clean install -Dmaven.test.skip=true
```
Comment (arvindkrishnakumar-okta) [inline .circleci/config.yml:8]: This configuration would actually skip the Unit tests also. We should not skip the UTs.

File: .circleci/config.yml [hunk 2]
```diff
@@ -20,25 +26,31 @@ jobs:
       - image: cimg/openjdk:8.0.345-node
     environment:
       JVM_OPTS: -Xmx3200m
+    resource_class: large
+    parallelism: 3
```
Comment (arvindkrishnakumar-okta) [inline .circleci/config.yml:30]: What does this setting `3` do?

Comment (karthikaselvaraj-okta) [reply .circleci/config.yml:30]: Parallelism:3 --> It creates 3 different containers to run the IT in parallel

**Review by arvindkrishnakumar-okta — CHANGES_REQUESTED** [review]
CircleCI build logs for Java 8, 11 & 17 all show that only 1 IT (IdpIT) is run for the below command:

```
    - run:
        command: |
          ClassName=$(circleci tests glob "integration-tests/src/test/groovy/com/okta/sdk/tests/it/**IT.groovy" | circleci tests split --split-by=filesize | grep groovy | xargs -n 1 basename | sed -e 's/\.groovy$//' | tr '\n' ',' | sed 's/,*$//g')
          echo $ClassName
          cd ./integration-tests
          mvn -Dtest=$ClassName test
```

image

**Review by vijetmahabaleshwar-okta — APPROVED** [review]

**Review by arvindkrishnakumar-okta — APPROVED** [review]

**Pull request merged**